data/core/units/humans/Mage_Arch.cfg

```diff
@@ -12,7 +12,8 @@
     experience=150
     level=3
     alignment=neutral
-    advances_to=Great Mage
```

Comment:
Guess, players cannot braindeadly spam this anymore in campaigns. So a good change.